dali/kernels/signal/dct/dct_gpu.cu

```diff
@@ -120,7 +159,8 @@ template <typename OutputType, typename InputType>
 DLL_PUBLIC void Dct1DGpu<OutputType, InputType>::Run(KernelContext &ctx,
                                                      const OutListGPU<OutputType> &out,
                                                      const InListGPU<InputType> &in,
-                                                     span<const DctArgs>, int) {
+                                                     span<const DctArgs>, int,
+                                                     span<const float> lifter_coeffs) {
```

Comment:
Is that a device pointer? If so, it should be marked as such.

Reply:
I've used InTensorGPU